tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ClientCredentialWithCertTest.cs

```diff
@@ -136,6 +163,60 @@ public async Task JsonWebTokenWithX509PublicCertSendCertificateOnBehalfOfTestAsy
                     .ConfigureAwait(false);
 
                 Assert.IsNotNull(result.AccessToken);
+
+                //Check user cache
+                Assert.AreEqual(1, app.UserTokenCacheInternal.Accessor.GetAllAccessTokens().Count());
+
+                //Clear cache
+                app.UserTokenCacheInternal.ClearMsalCache();
+            }
+        }
+
+        [TestMethod]
+        [Description("Test for client assertion with X509 public certificate using sendCertificate")]
+        public async Task JsonWebTokenWithX509PublicCertSendCertificateSilentTestAsync()
+        {
+            using (var harness = CreateTestHarness())
+            {
+                SetupMocks(harness.HttpManager, "https://login.microsoftonline.com/my-utid/");
+                var certificate = new X509Certificate2(
+                    ResourceHelper.GetTestResourceRelativePath("valid_cert.pfx"),
+                    MsalTestConstants.DefaultPassword);
+
+                var app = ConfidentialClientApplicationBuilder.Create(MsalTestConstants.ClientId)
+                                                              .WithAuthority(
+                                                                  new System.Uri("https://login.microsoftonline.com/my-utid"),
+                                                                  true).WithRedirectUri(MsalTestConstants.RedirectUri)
```

Comment:
move to new line

Reply:
wrapping before the . can make this look nicer

var app = CCABuilder
   .Create(xxx)
   .WithAuthority(new uri())
... etc.